tests/integration-tests-base/pom.xml

```diff
@@ -34,7 +34,7 @@
   <artifactId>integration-tests-base</artifactId>
   <packaging>pom</packaging>
 
-  <name>Apache Pulsar :: Tests :: Base module for Arquillian based integration tests</name>
+  <name>Apache Pulsar :: Tests :: Base module for TestContainers based integration tests</name>
```

Comment:
if you combine all integration modules into one integration module, I would prefer calling it "Apache Pulsar :: Tests :: Integration Tests". there is no need to call out "testcontainers". just like we used testng for testing, but we don't call out testng in any module's name or description.